app/views/docs/queries.phtml

```diff
@@ -0,0 +1,500 @@
+            <th>SDK Method Example</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Equal</td>
+            <td><code>Query.equal("title", ["Iron Man"])</code></td>
+            <td>Returns document if attribute is equal to any value in the provided array.</td>
+        </tr>
+        <tr>
+            <td>Not Equal</td>
+            <td><code>Query.notEqual("title", ["Iron Man"])</code></td>
+            <td>Returns document if attribute is not equal to any value in the provided array.</td>
+        </tr>
+        <tr>
+            <td>Less Than</td>
+            <td><code>Query.lessThan("score", 10)</code></td>
+            <td>Returns document if attribute is less than the provided value.</td>
+        </tr>
+        <tr>
+            <td>Less Than or Equal</td>
+            <td><code>Query.lessThanEqual("score", 10)</code></td>
+            <td>Returns document if attribute is less than or equal to the provided value.</td>
+        </tr>
+        <tr>
+            <td>Greater Than</td>
+            <td><code>Query.greaterThan("score", 10)</code></td>
+            <td>Returns document if attribute is greater than the provided value.</td>
+        </tr>
+        <tr>
+            <td>Greater Than or Equal</td>
+            <td><code>Query.greaterThanEqual("score", 10)</code></td>
+            <td>Returns document if attribute is greater than or equal to the provided value.</td>
+        </tr>
+        <tr>
+            <td>Between</td>
+            <td><code>Query.between("price", [5, 10])</code></td>
+            <td>Returns document if attribute value falls between the two values. The boundary values are inclusive.</td>
+        </tr>
+        <tr>
+            <td>Is Null</td>
+            <td><code>Query.isNull("name")</code></td>
+            <td>Returns documents where attribute value is null.</td>
+        </tr>
+        <tr>
+            <td>Is Not Null</td>
+            <td><code>Query.isNotNull("name")</code></td>
+            <td>Returns documents where attribute value is <b>not</b> null.</td>
+        </tr>
+        <tr>
+            <td>Starts With</td>
+            <td><code>Query.startsWith("name", ["Once upon a time"])</code></td>
+            <td>Returns documents if a string attributes starts with a substring.</td>
+        </tr>
+        <tr>
+            <td>Ends With</td>
+            <td><code>Query.endsWith("name", ["happily ever after."])</code></td>
+            <td>Returns documents if a string attributes ends with a substring.</td>
+        </tr>
+        <tr>
+            <td>Search</td>
+            <td><code>Query.search("text", "key words")</code></td>
+            <td>Searches string attributes for provided keywords. Applies to any string attribute with a full-text index.</td>
+        </tr>
+        <tr>
+            <td>Select</td>
+            <td><code>Query.select(["name", "title"])</code></td>
```

Comment:
Let's add a note somewhere, probably not here, that the select query can also be used with `getDocument` calls, not only `listDocuments`